scrapi/consumers/calpoly/README.md

```diff
@@ -0,0 +1,8 @@
+CalPoly
+=======
+
+Consumer for the Digital Commons @ CalPoly API for the SHARE project
```

Comment:
Yeah, these readmes probably would be better as docstrings. I have seen some people use extra readmes to point at additional documentation though, so a README in the consumers directory pointing at consumer documentation could be helpful. I wasn't able to find anything about best practices though.